src/figuresClasses.ts

```diff
@@ -1,19 +1,83 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+function notTriangle(side1: number, side2: number, side3: number) :boolean {
```

Comment:
```suggestion
function notTriangle(
  side1: number,
  side2: number,
  side3: number
) :boolean {
```
As for me, little bit difficult to read))